.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.0.0
+  hooks:
+    #- id: check-yaml
+    #  args: [--allow-multiple-documents]
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+    - id: check-symlinks
+    - id: mixed-line-ending
+
+- repo: https://github.com/jumanjihouse/pre-commit-hooks
```

Comment:
We should probably fork/vendor this, just a precautionary measure (supply chain attacks or if the repo goes away).

Reply:
fixed